tljh/installer.py

```diff
@@ -499,6 +504,17 @@ def main():
     ensure_node()
     ensure_jupyterhub_package(HUB_ENV_PREFIX)
     ensure_jupyterlab_extensions()
+
+    # Stop the http server with the loading page before traefik starts
+    if args.temporary_page:
+        try:
+            os.kill(int(args.temporary_page), signal.SIGINT)
+            # Remove the pid file and the temporary html page
+            os.remove('/index.html')
```

Comment:
We should put everything under `/var/run` or someplace, and not under `/`. This helps keep `/` clean!